### PR TITLE
[zh-cn] sync case-studies/daocloud

### PR DESCRIPTION
**Pull request by my-git9**
[zh-cn] sync case-studies/daocloud

### CODE REVIEW

File: content/zh-cn/case-studies/daocloud/index.html [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+<!--
+<h2>Solutions</h2>
+-->
+<h2>解决方案</h2>
+
+<!--
+<p>As the de facto standard for container orchestration, Kubernetes is undoubtedly the preferred container solution. Paco Xu, head of the Open Source and Advanced Development team at DaoCloud, stated, "Kubernetes is a fundamental tool in the current container ecosystem. Most services or applications are deployed and managed in Kubernetes clusters."</p>
+-->
+<p>Kubernetes 作为容器编排的事实标准，无疑是首选的容器解决方案。「DaoCloud 道客」架构师兼开源团队负责人徐俊杰（Paco）表示 “Kubernetes 是目前容器生态里面比较基础的一环，绝大多数服务都是基于 Kubernetes 部署的，应用绝大多数都是在 Kubernetes 集群中运行和管理。“</p>
```
Comment (windsonsea) [inline content/zh-cn/case-studies/daocloud/index.html:67]: Not sure whether it's suitable to show the real name in this article.

Comment (my-git9) [reply content/zh-cn/case-studies/daocloud/index.html:67]: My fault, sorry

Comment (my-git9) [reply content/zh-cn/case-studies/daocloud/index.html]: Can you review again, thanks!

File: content/zh-cn/case-studies/daocloud/index.html [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+<!--
+<p>For a large-scale securities firm, its business procedure efficiency was enhanced by 30%, and resource costs were lowered by about 35%.</p>
+-->
+<p>为某大型综合类券商搭建统一的云原生 PaaS 平台，使其业务流程效率提升 30%，资源成本节约 35% 左右；</p>
+
+<!--
+<p>With this product, Fullgoal Fund shortened its middleware deployment time from hours to minutes, improved middleware operation and maintenance capabilities by 50%, containerization by 60%, and resource utilization by 40%.</p>
+-->
+为富国基金打造基于 Kubernetes 的新一代云原生 PaaS 平台，将标准中间件部署时间从数小时缩短至数分钟，中间件运维能力提升 50%，容器化程度提升 60%，资源利用率提升 40%。
```
Comment (windsonsea) [inline content/zh-cn/case-studies/daocloud/index.html:172]:
```suggestion
<p>为富国基金打造基于 Kubernetes 的新一代云原生 PaaS 平台，将标准中间件部署时间从数小时缩短至数分钟，
中间件运维能力提升 50%，容器化程度提升 60%，资源利用率提升 40%。</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+<p>In addition, DaoCloud continue to practice its cloud native beliefs and contribute to the Kubernetes ecosystem by sharing source code of several excellent projects, including <a href="https://clusterpedia.io/">Clusterpedia</a>, <a href="https://github.com/kubean-io/kubean">Kubean</a>, <a href="https://github.com/cloudtty/cloudtty">CloudTTY</a>, <a href="https://github.com/klts-io/kubernetes-lts">KLTS</a>, <a href="https://merbridge.io/">Merbridge</a>, <a href="https://hwameistor.io/">HwameiStor</a>, <a href="https://github.com/spidernet-io/spiderpool">Spiderpool</a>, and <a href="https://github.com/kubernetes-sigs/kwok">KWOK</a>, on GitHub.</p>
+-->
+<p>此外，DaoCloud 也在坚持践行云原生信仰，持续回馈云原生社区，开源了 <a href="https://clusterpedia.io/">Clusterpedia</a>, <a href="https://github.com/kubean-io/kubean">Kubean</a>, <a href="https://github.com/cloudtty/cloudtty">CloudTTY</a>, <a href="https://github.com/klts-io/kubernetes-lts">KLTS</a>, <a href="https://merbridge.io/">Merbridge</a>, <a href="https://hwameistor.io/">HwameiStor</a>, <a href="https://github.com/spidernet-io/spiderpool">Spiderpool</a>, and <a href="https://github.com/kubernetes-sigs/kwok">KWOK</a> 等优秀项目，不断完善 Kubernetes 生态体系。</p>
```
Comment (windsonsea) [inline content/zh-cn/case-studies/daocloud/index.html:207]:
```suggestion
<p>此外，DaoCloud 也在坚持践行云原生信仰，持续回馈云原生社区，
开源了 <a href="https://clusterpedia.io/">Clusterpedia</a>、
<a href="https://github.com/kubean-io/kubean">Kubean</a>、
<a href="https://github.com/cloudtty/cloudtty">CloudTTY</a>、
<a href="https://github.com/klts-io/kubernetes-lts">KLTS</a>、
<a href="https://merbridge.io/">Merbridge</a>、
<a href="https://hwameistor.io/">HwameiStor</a>、
<a href="https://github.com/spidernet-io/spiderpool">Spiderpool</a>
和 <a href="https://github.com/kubernetes-sigs/kwok">KWOK</a>
等优秀项目，不断完善 Kubernetes 生态体系。</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+
+<!--
+<h2>Challenges</h2>
+-->
+<h2>挑战</h2>
+
+<!--
+<p><a href="https://www.daocloud.io/en/">DaoCloud</a>, founded in 2014, is an innovation leader in the field of cloud native. It boasts independent intellectual property rights of core technologies for crafting an open cloud platform to empower the digital transformation of enterprises.</p>
+-->
+<p>作为云原生领域的创新领导者，<a href="https://www.daocloud.io/en/">DaoCloud</a> 成立于 2014 年底，拥有自主知识产权的核心技术，致力于打造开放的云操作系统，为企业数字化转型赋能。</p>
```
Comment (windsonsea) [inline content/zh-cn/case-studies/daocloud/index.html:47]:
```suggestion
<p>作为云原生领域的创新领导者，<a href="https://www.daocloud.io/">DaoCloud</a> 成立于 2014 年底，
拥有自主知识产权的核心技术，致力于打造开放的云操作系统，为企业数字化转型赋能。</p>
```
这种 `<p></p>` 应该也可以参照 [k8s 风格指南](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#layout-format)折行吧

File: content/zh-cn/case-studies/daocloud/index.html [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+<!--
+<p>DaoCloud has been engaged in cloud native since its inception. As containerization is crucial for cloud native business, a cloud platform that does not have containers as infrastructure is unlikely to attract its potential users. Therefore, the first challenge confronting DaoCloud is how to efficiently manage and schedule numerous containers while maintaining stable connectivity between them.</p>
+-->
+<p>DaoCloud 从一开始就在做云原生。由于容器化对于云原生业务至关重要，没有容器作为基础设施的云平台不太可能吸引其潜在用户。因此，DaoCloud 面临的第一个挑战就是如何高效地管理和调度众多的容器，同时保持它们之间的稳定连接。</p>
```
Comment (windsonsea) [inline content/zh-cn/case-studies/daocloud/index.html:52]:
```suggestion
<p>DaoCloud 从诞生之日起就在致力于布道云原生。由于容器化对于云原生业务至关重要，
没有容器作为基础设施的云平台不太可能吸引其潜在用户。因此，DaoCloud
面临的第一个挑战就是如何高效地管理和调度众多的容器，同时保持它们之间的稳定连接。</p>
```

**Review by windsonsea — COMMENTED** [review]
Should we keep consistent with [localization-zh style guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#layout-format)?

File: content/zh-cn/case-studies/daocloud/index.html (hunk 5 above)
Comment (my-git9) [reply content/zh-cn/case-studies/daocloud/index.html:52]: I will try it later

File: content/zh-cn/case-studies/daocloud/index.html (hunk 4 above)
Comment (my-git9) [reply content/zh-cn/case-studies/daocloud/index.html:47]: I will try it later

Comment (my-git9) [reply content/zh-cn/case-studies/daocloud/index.html]: The preview seems good!

File: content/zh-cn/case-studies/daocloud/index.html [hunk 6]
```diff
@@ -0,0 +1,328 @@
...
+
+<!--
+<h2>Challenges</h2>
+-->
+<h2>挑战</h2>
+
+<!--
+<p><a href="https://www.daocloud.io/en/">DaoCloud</a>, founded in 2014, is an innovation leader in the field of cloud native. It boasts independent intellectual property rights of core technologies for crafting an open cloud platform to empower the digital transformation of enterprises.</p>
+-->
+<p>作为云原生领域的创新领导者，<a href="https://www.daocloud.io/">DaoCloud</a> 成立于 2014 年底，
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:47]:
```suggestion
<p>
作为云原生领域的创新领导者，<a href="https://www.daocloud.io/">DaoCloud</a> 成立于 2014 年底，
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+  拥有自主知识产权的核心技术，致力于打造开放的云操作系统，为企业数字化转型赋能。</p>
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:48]:
```suggestion
拥有自主知识产权的核心技术，致力于打造开放的云操作系统，为企业数字化转型赋能。
</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+  Kubernetes（v1.6.7）作为容器编排工具。此后，2018 年发布的 DaoCloud Enterprise 3.0
+  采用 Kubernetes 1.10 版本，2021 年发布的 DaoCloud Enterprise 4.0 采用 Kubernetes 1.18 版本。
+  2022 年发布的 DaoCloud Enterprise 5.0 支持 Kubernetes 1.23 至 1.26 版本。</p>
+
+<!--
+<p>Kubernetes served as an inseparable part of these four releases over six years, which speaks volumes about the fact that using Kubernetes in DaoCloud Enterprise was the right choice. DaoCloud has proven, through its own experience and actions, that Kubernetes is the best choice for container orchestration and that it has always been a loyal fan of Kubernetes.</p>
+-->
+<p>六年时间里发布的四个主要版本一直都在坚定不移地使用 Kubernetes，这足以说明当时的选择是正确的。
+  DaoCloud 用实际经验证明了 Kubernetes 是容器编排的最佳选择，也用自身行动证明了自己一直都是
+  Kubernetes 的忠实拥趸。</p>
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:148]: 译为“拥趸”，有点文绉绉的味道。

File: content/zh-cn/case-studies/daocloud/index.html [hunk 9]
```diff
@@ -0,0 +1,328 @@
...
+<p>Kubernetes helped our product and research teams realized automation of test, build, check, and release process, ensuring the quality of deliverables. It also helped build our smart systems of collaboration about product requirements & definition, multilingual product materials, debugging, and miscellaneous challenges, improving the efficiency of intra- and inter-department collaboration. </p>
+-->
+<p>Kubernetes 帮助我们的产研团队从 0 到 1 完成了从研发构建自动化，测试自动化，安全自动化，
+  发布自动化保证了软件交付质量，其次实现了智能化协作沟通，包括产品需求及定义体系、产品多语言协作体系、
+  产品缺陷修复协作体系、疑难杂症攻坚体系，极大的提升了产研同部门、跨部门的协作效率。</p>
+
+<!--
+<p>On the one hand, Kubernetes makes our products more performant and competitive. DaoCloud integrates relevant practices and technologies around Kubernetes to polish its flagship offering – DaoCloud Enterprise. The latest 5th version, released in 2022, covers application stores, application delivery, microservice governance, observability, data services, multi-cloud management, cloud-edge collaboration, and other functions. DaoCloud Enterprise 5.0 is an inclusive integration of cloud native technologies.</p>
+-->
+<p>在 Kubernetes 的助力下，DaoCloud  的产品性能更优，更具竞争力。DaoCloud 坚持以
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:175]:
```suggestion
<p>在 Kubernetes 的助力下，DaoCloud 的产品性能更优，更具竞争力。DaoCloud 坚持以
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 10]
```diff
@@ -0,0 +1,328 @@
...
+<p>DaoCloud deployed a Kubernetes platform for SPD Bank, improving its application deployment efficiency by 82%, shortening its delivery cycle from half a year to one month, and promoting its transaction success rate to 99.999%.</p>
+-->
+<p>DaoCloud 为上海浦发银行部署 Kubernetes 平台后，应用部署效率提升 82%，交付周期从半年缩短到一个月，
+  交易成功率达到 99.999%；</p>
+
+<!--
+<p>In terms of Sichuan Tianfu Bank, the scaling time was reduced from several hours to an average of 2 minutes, product iteration cycle was shortened from two months to two weeks, and application rollout time was cut by 76.76%.</p>
+-->
+<p>四川天府银行落地基于 Kubernetes 的云原生平台，将弹性响应时间由数小时大幅缩减到平均 2 分钟，
+  产品迭代周期从两个月缩短为两周，应用上线时间缩短 76.76%；</p>
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:190]:
```suggestion
  产品迭代周期从两个月缩短为两周，应用上线时间缩短 76.76%。</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 11]
```diff
@@ -0,0 +1,328 @@
...
+
+<!--
+<p>As for a joint-venture carmaker, its delivery cycle shortened from two months to one or two weeks, success rate of application deployment increased by 53%, and application rollout became ten times more efficient. In the case of a multinational retailer, application deployment issues were solved by 46%, and fault location efficiency rose by more than 90%.</p>
+-->
+<p>为某合资车企搭建基于 Kubernetes 的云原生平台后，将其交付周期从两个月缩短到一两周，应用部署成功率提升
+  53%，应用上线效率提高 24 倍；为某跨国零售集团部署基于 Kubernetes 的多个云原生平台模块，为其减少了
+  46% 的应用部署问题，将监控定位效率提升 90% 以上；</p>
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:197]:
```suggestion
  46% 的应用部署问题，将监控定位效率提升 90% 以上。</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 12]
```diff
@@ -0,0 +1,328 @@
...
+
+<!--
+<p>For a large-scale securities firm, its business procedure efficiency was enhanced by 30%, and resource costs were lowered by about 35%.</p>
+-->
+<p>为某大型综合类券商搭建统一的云原生 PaaS 平台，使其业务流程效率提升 30%，资源成本节约 35% 左右；</p>
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:202]:
```suggestion
<p>为某大型综合类券商搭建统一的云原生 PaaS 平台，使其业务流程效率提升 30%，资源成本节约 35% 左右。</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 13]
```diff
@@ -0,0 +1,328 @@
...
+<p>With this product, Fullgoal Fund shortened its middleware deployment time from hours to minutes, improved middleware operation and maintenance capabilities by 50%, containerization by 60%, and resource utilization by 40%.</p>
+-->
+<p>为富国基金打造基于 Kubernetes 的新一代云原生 PaaS 平台，将标准中间件部署时间从数小时缩短至数分钟，
+  中间件运维能力提升 50%，容器化程度提升 60%，资源利用率提升 40%。
+
+<!--
+<p>On the other hand, our product development is also based on Kubernetes. DaoCloud deployed Gitlab based on Kubernetes and established a product development process of "Gitlab -> PR -> Auto Tests -> Builds & Releases", which significantly improved our development efficiency, reduced repetitive tests, and realized automatic release of applications. This approach greatly saves operation and maintenance costs, enabling technicians to invest more time and energy in product development to offer better cloud native products.</p>
+-->
+<p>另一方面，DaoCloud 自身的产品研发工作也是基于 Kubernetes 进行的。公司基于 Kubernetes
+  部署了 Gitlab，形成了 “Gitlab —> PR —> 自动化测试 —> 构建发布“ 的产品开发流程，显著提升了开发效率，
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:214]:
```suggestion
  部署了 Gitlab，形成了 “Gitlab —> PR —> 自动化测试 —> 构建发布“的产品开发流程，显著提升了开发效率，
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 14]
```diff
@@ -0,0 +1,328 @@
...
+{{< case-studies/quote
+  image="/images/case-studies/daocloud/banner4.jpg"
+  author="Paco Xu, Header of Open Source & Advanced Development Team, DaoCloud"
+>}}
+"Our developers actively contribute to open source projects and build technical expertise. DaoCloud has established a remarkable presence in the Kubernetes and Istio communities."
+{{< /case-studies/quote >}}
+-->
+{{< case-studies/quote
+  image="/images/case-studies/daocloud/banner4.jpg"
+  author="Paco Xu，DaoCloud 架构师/ 开源 & AD 团队负责人"
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:228]:
```suggestion
  author="Paco Xu，DaoCloud 开源和高级开发团队负责人"
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 15]
```diff
@@ -0,0 +1,328 @@
...
+-->
+<p>DaoCloud 深度参与贡献 Kubernetes 等多项云原生开源项目，在云原生开源社区中的参与度、贡献度持续增长。
+  在过去一年里，DaoCloud 在 Kubernetes 的开源榜单累计贡献度位居全球第三（基于 Stackalytics 网站 2023/06/13 的数据）。</p>
+
+<!--
+<p>In August 2022, Kubernetes officially organized an interview with community contributors, and four outstanding contributors from the Asia-Pacific region were invited. Half of them came from DaoCloud, namely <a href="https://github.com/wzshiming">Shiming Zhang</a> and <a href="https://github.com/pacoxu">Paco Xu</a>. Both are Reviewers of SIG Node. Furthermore, at the KubeCon + CloudNative North America 2022, <a href="https://github.com/kerthcet">Kante Yin</a> from DaoCloud won the 2022 Contributor Award of Kubernetes.</p>
+-->
+<p>在 2022 年 8 月由 Kubernetes 官方组织的社区贡献者访谈活动中，接见了来自亚太地区的 4 位优秀贡献者，
+  其中 Shiming Zhang 和 Paco Xu 都来自 DaoCloud ，二人均是 SIG Node 的 Reviewer。此外在
+  2022 Kubecon 北美站上， DaoCloud 的 Kante Yin 荣获 Kubernetes 2022 年度贡献者奖。</p>
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:245]:
```suggestion
  2022 Kubecon 北美站上，DaoCloud 的 Kante Yin 荣获 Kubernetes 2022 年度贡献者奖。</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 16]
```diff
@@ -0,0 +1,328 @@
...
+<p>DaoCloud is deeply involved in contributing to Kubernetes and other cloud native open source projects. Our participation and contributions in these communities continue to grow. In the year of 2022, DaoCloud was ranked third globally in terms of cumulative contribution to Kubernetes (data from Stackalytics as of January 5, 2023).</p>
+-->
+<p>DaoCloud 深度参与贡献 Kubernetes 等多项云原生开源项目，在云原生开源社区中的参与度、贡献度持续增长。
+  在过去一年里，DaoCloud 在 Kubernetes 的开源榜单累计贡献度位居全球第三（基于 Stackalytics 网站 2023/06/13 的数据）。</p>
+
+<!--
+<p>In August 2022, Kubernetes officially organized an interview with community contributors, and four outstanding contributors from the Asia-Pacific region were invited. Half of them came from DaoCloud, namely <a href="https://github.com/wzshiming">Shiming Zhang</a> and <a href="https://github.com/pacoxu">Paco Xu</a>. Both are Reviewers of SIG Node. Furthermore, at the KubeCon + CloudNative North America 2022, <a href="https://github.com/kerthcet">Kante Yin</a> from DaoCloud won the 2022 Contributor Award of Kubernetes.</p>
+-->
+<p>在 2022 年 8 月由 Kubernetes 官方组织的社区贡献者访谈活动中，接见了来自亚太地区的 4 位优秀贡献者，
+  其中 Shiming Zhang 和 Paco Xu 都来自 DaoCloud ，二人均是 SIG Node 的 Reviewer。此外在
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:244]:
```suggestion
  其中 Shiming Zhang 和 Paco Xu 都来自 DaoCloud，二人均是 SIG Node 的 Reviewer。此外在
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 17]
```diff
@@ -0,0 +1,328 @@
...
+<p>企业用户需要的是一个全局最优解，这个最优解可以理解为是涵盖多云编排、信创异构、
+  应用交付、可观测性、云边协同、微服务治理、应用商店、数据服务等能力的最大公约数。
+  在如今的云原生生态体系里，这些功能都离不开 Kubernetes 作为底层的容器编排技术。
+  这就意味着 DaoCloud 在寻找数字世界最优解的过程中也离不开 Kubernetes，
+  未来的产品研发也将继续以 Kubernetes 为基础。</p>
+
+<!--
+<p>Kubernetes training and promotion activities have always been attached great importance in DaoCloud. In 2017, the company took the lead in passing CNCF's Certified Kubernetes Conformance Program through its featured product — DaoCloud Enterprise. In 2018, it became a CNCF-certified Kubernetes service provider and training partner.</p>
+-->
+<p>此外，DaoCloud 一直致力于 Kubernetes 的培训、推广活动。2017年，公司凭借核心产品云原生应用云平台 DaoCloud Enterprise 成为全球首批通过 CNCF Kubernetes 兼容性认证的厂家。2018年，公司成为 CNCF 认证的 Kubernetes 服务提供商，并成为全球首批获得CNCF官方认证的 Kubernetes培训合作伙伴，全面拥抱 Kubernetes 技术生态。</p>
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:319]:
```suggestion
<p>
此外，DaoCloud 一直致力于 Kubernetes 的培训、推广活动。
2017 年，公司凭借核心产品云原生应用云平台 DaoCloud Enterprise
成为全球首批通过 CNCF Kubernetes 兼容性认证的厂家。
2018 年，公司成为 CNCF 认证的 Kubernetes 服务提供商，
并成为全球首批获得CNCF官方认证的 Kubernetes培训合作伙伴，
全面拥抱 Kubernetes 技术生态。
</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 18]
```diff
@@ -0,0 +1,328 @@
...
+
+<!--
+<p>On November 18, 2022, the "Kubernetes Community Days" event was successfully held in Chengdu, organized by CNCF, DaoCloud, Huawei Cloud, Sichuan Tianfu Bank, and OPPO. The event brought together end-users, contributors, and technical experts from open-source communities to share best practices and innovative ideas about Kubernetes and cloud native. In the future, DaoCloud will continue to contribute to Kubernetes projects, and expand the influence of Kubernetes through project training, community contributions and other activities.</p>
+-->
+<p>2022 年 11 月 18 日，由 CNCF 和 DaoCloud 、华为云、四川天府银行、
```
Comment (howieyuen) [inline content/zh-cn/case-studies/daocloud/index.html:324]:
```suggestion
<p>
2022 年 11 月 18 日，由 CNCF 和 DaoCloud、华为云、四川天府银行、
```

**Review by howieyuen — COMMENTED** [review]
In the Chinese translation style guide, there are style descriptions about line breaks and spaces mixed in zh and en.
Although the preview page is fine, it is not an easy task for code reviewer.
Please refresh the whole page, thanks.

**Comment by Michelle951** [comment]
This Chinese version is exactly identical with the blog I wrote and published on DaoCloud doc website in about Feb 2023. See: <https://github.com/DaoCloud/DaoCloud-docs/pull/869>
@my-git9

**Comment by windsonsea** [comment]
hmmm, it's not suitable to close if reviewers has put many efforts here. Respect comes from each other.

**Comment by my-git9** [comment]
> hmmm, it's not suitable to close if reviewers has put many efforts here. Respect comes from each other.

sorry for that, I will continue to work on this

**Comment by windsonsea** [comment]
@my-git9 You can have a try to [co-authored](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) with @Michelle951. Also, respect the above comments.

**Comment by my-git9** [comment]
> co-authored

OK, let me know how to do it, thanks!

**Comment by my-git9** [comment]
@Michelle951  Hi, I set the [co-authored](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) with you, do you support this form and can you check if the information is correct?
image

File: content/zh-cn/case-studies/daocloud/index.html [hunk 19]
```diff
@@ -0,0 +1,372 @@
...
+>}}
+"Kubernetes 是将我们的产品改进为世界一流软件的基石。"
+{{< /case-studies/quote >}}
+
+<!--
+<p>Kubernetes helped our product and research teams realized automation of test, build, check, and release process, ensuring the quality of deliverables. It also helped build our smart systems of collaboration about product requirements & definition, multilingual product materials, debugging, and miscellaneous challenges, improving the efficiency of intra- and inter-department collaboration. </p>
+-->
+<p>Kubernetes 帮助我们的产研团队从 0 到 1 完成了从研发构建自动化，测试自动化，安全自动化，
+  发布自动化保证了软件交付质量，其次实现了智能化协作沟通，包括产品需求及定义体系、产品多语言协作体系、
+  产品缺陷修复协作体系、疑难杂症攻坚体系，极大的提升了产研同部门、跨部门的协作效率。</p>
```
Comment (windsonsea) [inline content/zh-cn/case-studies/daocloud/index.html:190]:
```suggestion
<p>
Kubernetes 帮助我们的产研团队从 0 到 1 完成了从研发构建自动化，测试自动化，安全自动化，
发布自动化保证了软件交付质量，其次实现了智能化协作沟通，包括产品需求及定义体系、产品多语言协作体系、
产品缺陷修复协作体系、疑难杂症攻坚体系，极大的提升了产研同部门、跨部门的协作效率。
</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 20]
```diff
@@ -0,0 +1,372 @@
...
+应用交付、可观测性、云边协同、微服务治理、应用商店、数据服务等能力的最大公约数。
+在如今的云原生生态体系里，这些功能都离不开 Kubernetes 作为底层的容器编排技术。
+这就意味着 DaoCloud 在寻找数字世界最优解的过程中也离不开 Kubernetes，
+未来的产品研发也将继续以 Kubernetes 为基础。
+</p>
+
+<!--
+<p>Kubernetes training and promotion activities have always been attached great importance in DaoCloud. In 2017, the company took the lead in passing CNCF's Certified Kubernetes Conformance Program through its featured product — DaoCloud Enterprise. In 2018, it became a CNCF-certified Kubernetes service provider and training partner.</p>
+-->
+<p>此外，DaoCloud 一直致力于 Kubernetes 的培训、推广活动。2017年，公司凭借核心产品云原生应用云平台 DaoCloud Enterprise 成为全球首批通过 CNCF Kubernetes 兼容性认证的厂家。2018年，公司成为 CNCF 认证的 Kubernetes 服务提供商，并成为全球首批获得CNCF官方认证的 Kubernetes培训合作伙伴，全面拥抱 Kubernetes 技术生态。</p>
```
Comment (windsonsea) [inline content/zh-cn/case-studies/daocloud/index.html:361]:
```suggestion
<p>
此外，DaoCloud 一直致力于 Kubernetes 的培训、推广活动。2017 年，公司凭借核心产品云原生应用云平台
DaoCloud Enterprise 成为全球首批通过 CNCF Kubernetes 兼容性认证的厂家。2018年，公司成为 CNCF
认证的 Kubernetes 服务提供商，并成为全球首批获得CNCF官方认证的 Kubernetes培训合作伙伴，全面拥抱 Kubernetes 技术生态。
</p>
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 21]
```diff
@@ -0,0 +1,372 @@
...
+>}}
+“随着 DaoCloud Enterprise 越来越强大，客户覆盖面越来越广，有些客户需要使用 Kubernetes
+而不是 Swarm 进行应用编排。Kubernetes 作为一套功能强大的应用编排系统，有着强大的社区支持，
+也受到很多大公司的青睐。我们作为提供商需要满足用户的需求。”
+{{< /case-studies/quote >}}
+
+<!--
+<p>DaoCloud was founded to help traditional enterprises move their applications to the cloud and realize digital transformation. The first product released after the company's establishment, DaoCloud Enterprise 1.0, is a Docker-based container engine platform that can easily build images and run them in containers.</p>
+-->
+<p>DaoCloud 成立的初衷就是帮助传统企业进行数字化转型，实现应用上云。公司成立之后发布的首秀产品
```
Comment (windsonsea) [inline content/zh-cn/case-studies/daocloud/index.html:129]:
```suggestion
<p>
DaoCloud 成立的初衷就是帮助传统企业进行数字化转型，实现应用上云。公司成立之后发布的首秀产品
```

File: content/zh-cn/case-studies/daocloud/index.html [hunk 22]
```diff
@@ -0,0 +1,372 @@
...
+  DaoCloud Enterprise 1.0 便是一款基于 Docker 的容器引擎平台，可以轻松打包构建镜像并运行容器。</p>
```
Comment (windsonsea) [inline content/zh-cn/case-studies/daocloud/index.html:130]:
```suggestion
DaoCloud Enterprise 1.0 便是一款基于 Docker 的容器引擎平台，可以轻松打包构建镜像并运行容器。
</p>
```

**Review by windsonsea — COMMENTED** [review]
Pick nits: It can be consistent from top to bottom in an editing view:

```
<p>
A para is here.
</p>
```
The [preview](https://deploy-preview-41645--kubernetes-io-main-staging.netlify.app/zh-cn/case-studies/daocloud/) looks good now.

**Comment by windsonsea** [comment]
Thanks. IMO, it's time to merge now.

/lgtm

**Comment by Michelle951** [comment]
> @Michelle951 Hi, I set the [co-authored](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) with you, do you support this form and can you check if the information is correct? image

LGTM. Thanks

**Pull request merged**